src/component/mxgraph/config/StyleConfigurator.ts

```diff
@@ -298,6 +298,10 @@ export default class StyleConfigurator {
         if (markers.length > 0) {
           styleValues.set(StyleIdentifier.BPMN_STYLE_MARKERS, markers.join(','));
         }
+      } else if (bpmnElement.kind === ShapeBpmnElementKind.POOL) {
+        // mxConstants.STYLE_HORIZONTAL is for the label
+        // In BPMN, isHorizontal is for the Pool
+        styleValues.set(mxConstants.STYLE_HORIZONTAL, bpmnCell.isHorizontal ? '0' : '1');
```

Comment:
:question: why not putting a boolean like we do for the lane default style?
`style[mxConstants.STYLE_HORIZONTAL] = false;`
This is clearer and more understandable. We previously did it when we have a dedicated style entry for the activity `isExpanded`
But maybe this is a mxGraph constraint?

Reply:
I tried to do this, but it's not working.
As we need to convert boolean as string, the final style will be "horizontal=false". But MxGraph wants "horizontal=0".

Reply:
ok, fair enough, let keep this as it is